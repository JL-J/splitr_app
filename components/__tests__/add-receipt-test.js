import 'react-native';
import React from 'react';
import AddReceipt from '../addReceipt.js';

import renderer from 'react-test-renderer';

describe('<AddReceipt />', () => {

  it('renders AddReceipt page correctly', () => {
    const snap = renderer.create(
      <AddReceipt />
      ).toJSON();
    expect(snap).toMatchSnapshot();
  });

});
