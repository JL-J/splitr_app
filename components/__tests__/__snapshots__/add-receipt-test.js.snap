// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddDishes /> renders AddReceipt page correctly 1`] = `
  <View>
    <View>
      <Text>Add your Dish:</Text>
      <NameInput />
      <PriceInput />
      <SubmitButton />
    </View>
  </View>
`;
